hazelcast/src/main/java/com/hazelcast/nio/tcp/WriteHandler.java

```diff
@@ -312,4 +310,8 @@ public void run() {
             EmptyStatement.ignore(e);
         }
     }
+
+    public IOSelector getTargetIOSelector() {
```

Comment:
Would something with 'owner' not make more sense? You are currently migrating ownership of the WriteHandler from one selector to the next.